### ISSUE TITLE
Slide right opens period filter but doesn't work

### ISSUE THREAD

**Issue by walter-juan** [mvarnagiris/financius#60]
A period menu is opened if you slide right on any screen but it doesn't work, when you set one date the other date is removed automatically.

Maybe is a feature for remove, or show on specific screens?

![device screen](https://f.cloud.github.com/assets/4141614/1894832/9873168c-7b00-11e3-82c6-5237134d0b69.png)

**Comment by mvarnagiris** [comment]
It's work in progress :) I was hoping noone will find this :P

**Comment by GreenFomo** [comment]
@mvarnagiris Hey there! that panel.. it's quite easy to stumble upon, I instinctively swipe left and right whenever I install a new app ^_^

anyway, do you have a timeline for when this would be implemented? I'd really like to "switch" to Financius (using another $ tracking app atm, but lacks some features Financius has), but filtering for the current month is what would make this app 100% complete (it could/should have multiple "shortcuts" to "Current month", "Previous month", "Previous 3 months", and custom start + end)

Also, offtopic, but I'd like to be able to "hide" the "time" field when adding a transaction.. it simply irks me, my OCD kicks in for whatever reason..

Also, what will happen once I have, say, few hundreds transactions across multiple months and I go to the Transactions section? Will I see all those transactions at once? 

anyway, TLDR: when do you think you'd implement this period filter? 

Thanks!

**Comment by assoc** [comment]
Maybe, it is worth to use a bunch of grouping by month (show current as opened; hide another, but show titles of months and open at tap), and filter by category. I have for the year 800+ transactions, it's very long scroll-list.

**Comment by mvarnagiris** [comment]
Duplicate #20
